var/spack/repos/builtin/packages/gsl/package.py

```diff
@@ -55,3 +55,6 @@ def configure_args(self):
             configure_args.append("CBLAS_LIBS=%s" % self.spec["blas"].libs.ld_flags)
 
         return configure_args
+
+    def setup_run_environment(self, env):
+        env.set("GSL_ROOT_DIR", self.prefix)
```

Comment:
Can you add a brief comment above this line that this helps when using CMake and needing to disambiguate among different GSL installations?

Reply:
Done